geetools/ee_image.py

```diff
@@ -1678,6 +1678,72 @@ def fromList(images: ee.List | list[ee.Image]) -> ee.Image:
         ic = ee.ImageCollection.fromImages(images)
         return ic.toBands().rename(bandNames)
 
+    def classToBands(self, class_info: dict, band: str | int = 0) -> ee.Image:
+        """Convert each class into a separate binary mask.
+
+        Args:
+            class_info: class information.
+            band: band that contains class information
+
+        Example:
+            .. code-block:: python
```

Comment:
Let's try to use the jupyter-execute directive to see something here. you will need to use the documentation initialize parameter and display the image with `plot` I think it would be nice for the user to see what it does in the documentation.